### PR TITLE
AAP-30113 Emphasize correct decision environment usage in docs

### PR DESCRIPTION
**Pull request by jself-sudoku**
Updated the "Building a custom decision environment for Event-Driven Ansible" topic, downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc

### CODE REVIEW

File: downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc [hunk 1]
```diff
@@ -17,12 +17,20 @@ You can create a custom decision environment for {EDAName} that provides a custo
...
-Red Hat recommends using `de-minimal` as the base image with {Builder} to build your custom decision environments.
+Use `de-minimal` as the base image with {Builder} to build your custom decision environments.
+====
+
+[IMPORTANT]
+====
+* Use the correct {EDAcontroller} decision environment in {PlatformNameShort} to prevent rulebook activation failure.
+
+** If you want to connect {EDAcontroller} to {PlatformNameShort} 2.4, you must use link:https://catalog.redhat.com/software/containers/ansible-automation-platform-24/de-supported-rhel9/6449642e3993031ccdf4cf19[{PlatformNameShort} 2.4 supported decision environment]
+** If you want to connect {EDAcontroller} to {PlatformNameShort} {PlatformVers}, you must use link:https://catalog.redhat.com/software/containers/ansible-automation-platform-25/de-supported-rhel9/650a5674ad524b664b693729[{PlatformNameShort} {PlatformVers} supported execution environment]
```
Comment (zkayyali812) [inline downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc:33]: We should use the correct URLs here, from registry.redhat.io

File: downstream/modules/eda/ref-eda-controller-install-builder.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[id="eda-controller-install-builder"]
```
Comment (jself-sudoku) [inline downstream/modules/eda/ref-eda-controller-install-builder.adoc:1]: To reviewers: This is a new file that duplicates the same "Install ansible-builder" topic found in the controller docs.

**Review by zkayyali812 — APPROVED** [review]

**Review by jshimkus-rh — APPROVED** [review]

File: downstream/modules/eda/ref-eda-controller-install-builder.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+= Installing ansible-builder
+
+To build images, you must have Podman or Docker installed, along with the `ansible-builder` Python package. 
+
+The `--container-runtime` option must correspond to the Podman or Docker executable you intend to use.
+
+When building a decision environment image, it must support the architecture that {PlatformNameShort} is deployed with.
+
+For more information, see link:https://ansible.readthedocs.io/projects/builder/en/latest/#quickstart-for-ansible-builder[Quickstart for Ansible Builder] or link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/creating_and_using_execution_environments/index[Creating and consuming execution environments].
```
Comment (dcdacosta) [inline downstream/modules/eda/ref-eda-controller-install-builder.adoc:11]: Suggest using the link URL for the builder guide:

link:{LinkBuilder}.

**Review by dcdacosta — COMMENTED** [review]
One minor fix needed, otherwise, LGTM!

File: downstream/modules/eda/ref-eda-controller-install-builder.adoc (hunk 3 above)
Comment (jself-sudoku) [reply downstream/modules/eda/ref-eda-controller-install-builder.adoc:11]: Thanks, @dcdacosta. Correction in progress.

**Review by dcdacosta — APPROVED** [review]
LGTM!

File: downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc [hunk 4]
```diff
@@ -16,13 +16,16 @@ You can create a custom decision environment for {EDAName} that provides a custo
 
 .Procedure
 
-* Add the `de-supported` decision environment. 
-This image is built from a base image provided by Red Hat called `de-minimal` at link:https://catalog.redhat.com/software/containers/ansible-automation-platform-24/de-supported-rhel9/6449642e3993031ccdf4cf19[Ansible Automation Platform supported decision environment].
+* Use `de-minimal` as the base image with {Builder} to build your custom decision environments.
```
Comment (bzwei) [inline downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc:19]: It is confusing to user the name of base image in this document. Here and the examples use `de-minimal-rhel9`, while in the IMPORTANT section `de-supported-rhel9` is used. Which base image should be used `de-minimal` or `de-supported`?

Comment (jself-sudoku) [reply downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc:19]: Hi @bzwei . Thank you for your comments. By the time you were posting your message, we had move forward with our publishing process. I believe you ask a valid question and I attempted to make that clear in the content below letting the user know that they would use the `de-minimal` image that is provided in the registry.redhat.io by the `de-supported` images. I was confused initially, but thought tweaking the sentence below made it clearer that what the user should use is `de-minimal`. We can discuss this further, if necessary, in the "parking lot" portion of our stand-up, Wednesday, Nov. 13th.

**Pull request merged**